Yakymenko_Task_1.py

```diff
@@ -0,0 +1,8 @@
+# Multiparadigmatic  programming languages:Task_1
+# Yakymenko Semen:IKM-221a
+
+print('Multiparadigmatic  programming languages:Task_1\nSemen Yakymenko:IKM-221a')
+
+print((('Vladislav' + ', ') * 44) + 'Vladislav')
```

Comment:
why have you applied such concatenation? it's a regular string
'Vladislav' + ', ' ==> 'Vladislav, '

Reply:
also please investigate and try to understand example:
print((('Vladislav, ') * 45)[:-2])

Reply:
if you need detailed explanation feel free to ask me